docs/networking.md

```diff
@@ -55,8 +55,8 @@ scope driver.
 As you can see here, the ID is the same on the two nodes, because it's the same
 network.
 
-If you want to want to create a local scope network (for example with the bridge
-driver) you should use `<node>/<name>` otherwise your network will be created on a
+If you want to create a local scope network (for example with the bridge
```

Comment:
'locally scoped network'? 'network with local scope'? Do either of those read better for anyone? Maybe I'm parsing it wrong, but it sounds weird to me.